{
  return ((containedException == this) ? null : containedException);
}
